gsy_framework/constants_limits.py

```diff
@@ -185,7 +184,13 @@ class WindSettings:
 
     class HeatPumpSettings:
         """Default values for the heat pump."""
+        # range limits
+        MAX_POWER_RATING_KW_LIMIT = RangeLimit(0, sys.maxsize)
+        TEMP_C_LIMIT = RangeLimit(-sys.maxsize, sys.maxsize)
```

Comment:
Let's stay in the boundaries of physics and use the absolute zero and the temperature of the sun as boundaries ;-) 
 ```suggestion
        TEMP_C_LIMIT = RangeLimit(-273.15, 6000)
```